cluster-state-service/internal/features/e2e_stream_tasks.feature

```diff
@@ -0,0 +1,8 @@
+@e2e @task @stream-tasks
+Feature: Stream Tasks
+
+  Scenario: Get task stream
+    When I start streaming all task events
+    And I start 1 task in the ECS cluster
+    Then the stream tasks response contains at least 1 task
+    And the stream tasks response contains the task stopped
```

Comment:
Do we have to wait for the stopped event? The test may take too long. Can't we just wait for running?